requirements.txt

```diff
@@ -4,9 +4,21 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-aiohttp==2.0.7
-async-timeout==1.2.0      # via aiohttp
-chardet==3.0.2            # via aiohttp
-multidict==2.1.4          # via aiohttp, yarl
-sqlalchemy==1.1.9
-yarl==0.10.0              # via aiohttp
+#aiohttp
+#sqlalchemy
+#slacker-asyncio
+#
+# => produced ... why aiohttp==0.21.6 ????
```

Comment:
If it works with the newest aiohttp, then maybe `pip install --no-deps` will help --- all dependencies are already in the `requirements.txt` file anyway.